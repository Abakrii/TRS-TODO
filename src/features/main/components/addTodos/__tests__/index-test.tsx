import React from 'react';
import renderer from 'react-test-renderer';
import AddTodos from '../';

test('renders correctly', () => {
    const tree = renderer.create(
        <AddTodos
            addTodoHandler={() => console.log('add todo is working')}
        />).toJSON();
    expect(tree).toMatchSnapshot();
});


