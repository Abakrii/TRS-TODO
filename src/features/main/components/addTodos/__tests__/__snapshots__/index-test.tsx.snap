// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="codes.common.todoPlaceH"
    rejectResponderTermination={true}
    style={
      Object {
        "borderBottomColor": "#DDD",
        "borderBottomWidth": 1,
        "marginBottom": 10,
        "paddingHorizontal": 8,
        "paddingVertical": 6,
      }
    }
    underlineColorAndroid="transparent"
  />
  <View
    accessibilityRole="button"
    accessibilityState={
      Object {
        "disabled": true,
      }
    }
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
          Object {},
        ]
      }
    >
      <Text
        disabled={true}
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
            Object {
              "color": "coral",
            },
            Object {
              "color": "#cdcdcd",
            },
          ]
        }
      >
        codes.common.addTodo
      </Text>
    </View>
  </View>
</View>
`;
